explorer/app/viewModelBuilders/catalog/altos-labs-catalog/common/viewModelBuilders.ts

```diff
@@ -51,6 +51,21 @@ export const buildDOI = (
   };
 };
 
+/**
```

Comment:
Relocated method for alphabetical consistency (updated method name to `buildExperiment` from `buildTitle`).